specs/cli/client-edit.md

```diff
@@ -10,19 +10,19 @@ Once the `kiota-config.json` file and the API Manifest are updated, the code gen
 
 ## Parameters
 
-| Parameters | Required | Example | Description |
-| -- | -- | -- | -- |
-| `--client-name \| --cn` | Yes | GraphClient | Name of the client. Unique within the parent API. |
-| `--openapi \| -d` | No | https://aka.ms/graph/v1.0/openapi.yaml | The location of the OpenAPI description in JSON or YAML format to use to generate the SDK. Accepts a URL or a local path. |
-| `--include-path \| -i` | No | /me/chats#GET | A glob pattern to include paths from generation. Accepts multiple values. Defaults to no value which includes everything. |
-| `--exclude-path \| -e` | No | \*\*/users/\*\* | A glob pattern to exclude paths from generation. Accepts multiple values. Defaults to no value which excludes nothing. |
-| `--language \| -l` | No | csharp | The target language for the generated code files or for the information. |
-| `--namespace-name \| -n` | No | Contoso.GraphApp | The namespace of the client class. Defaults to `Microsoft.Graph`. |
-| `--backing-store \| -b` | No | | Defaults to `false` |
-| `--exclude-backward-compatible \| --ebc` | No |  | Whether to exclude the code generated only for backward compatibility reasons or not. Defaults to `false`. |
-| `--structured-media-types \| -m` | No | `application/json` |Any valid media type which will match a request body type or a response type in the OpenAPI description. Default are documented [here](https://learn.microsoft.com/en-us/openapi/kiota/using#--structured-mime-types--m). |
-| `--skip-generation \| --sg` | No | true | When specified, the generation would be skipped. Defaults to false. |
-| `--output \| -o` | No | ./generated/graph/csharp | The output directory or file path for the generated code files. Defaults to `./output`. |
+| Parameters | Required | Example | Description | Telemetry | 
+| -- | -- | -- | -- | -- |
+| `--client-name \| --cn` | Yes | graphDelegated | Name of the client. Unique within the parent API. If not provided, defaults to --class-name or its default. | Yes, without its value |
+| `--openapi \| -d` | No | https://aka.ms/graph/v1.0/openapi.yaml | The location of the OpenAPI description in JSON or YAML format to use to generate the SDK. Accepts a URL or a local path. | Yes, without its value |
+| `--include-path \| -i` | No | /me/chats#GET | A glob pattern to include paths from generation. Accepts multiple values. Defaults to no value which includes everything. | Yes, without its value |
+| `--exclude-path \| -e` | No | \*\*/users/\*\* | A glob pattern to exclude paths from generation. Accepts multiple values. Defaults to no value which excludes nothing. | Yes, without its value |
+| `--language \| -l` | No | csharp | The target language for the generated code files or for the information. | Yes |
+| `--namespace-name \| -n` | No | Contoso.GraphApp | The namespace of the client class. Defaults to `Microsoft.Graph`. | Yes, without its value |
+| `--backing-store \| -b` | No | | Defaults to `false` | Yes, without its value |
```

Comment:
If you specify it, it's true, so that's why. Like here: https://learn.microsoft.com/en-us/openapi/kiota/using#--backing-store--b

Reply:
Yeah, if people are using --backing-store we should get its value. So I believe the telemetry column should be "Yes" rather than "Yes, without its value", right?

Reply:
It's back to Yes! Absolutely!